pkg/kn/commands/version/version.go

```diff
@@ -26,19 +28,22 @@ var BuildDate string
 var GitRevision string
 var ServingVersion string
```

Comment:
tbh, I don't it makes sense to have this variable fed in from the outside when we have to update the content of this source anyway for each serving update. I would just print out the supported and used the version for this client from a fixed string which might be updated over time.

Later on, we can query the server for the supported versions (from the CRD) and print this, too. Ideally we would also support multiple versions and depending on the server support choose one the version that we support, too (currently, it's just `v1alpha1` but we should jump on `v1beta1` or even `v1` soonish)

Reply:
I was inclined towards hardcoding this too for the reason that we have to update this file after serving dep bump. Thanks for the +1, will update.